blocks/tabs-icons/tabs-icons.js

```diff
@@ -0,0 +1,18 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+import { div } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  block.classList.add('calcite-mode-dark');
+  document.querySelector('.tabs-container').classList.add('calcite-mode-dark');
+  document.querySelector('.tabs-container').classList.remove('calcite-mode-light');
```

Comment:
As discussed modes shouldn't be hardcoded, in general we expect that modes will be set at the body level, you can test by setting document-level metadata with Theme => dark/light